locales/templates/app.pot

```diff
@@ -404,3 +404,29 @@ msgctxt "exit_firefox_a11y"
 msgid "Exit %1$s"
 msgstr ""
 
+#. Hint Strings
+#. Hint text displayed when the overlay is open.      $IMAGE will be replaced
+#. with the image of a hardware back button.      When this button is pressed,
+#. the overlay will close and the user will be returned to the browser.
+msgctxt "hint_press_back_to_close_overlay"
+msgid "Press $IMAGE to return"
+msgstr ""
+
+msgctxt "hint_press_back_to_close_overlay_a11y"
+msgid "Press the back button to return"
+msgstr ""
+
+#. Hint text displayed when the browser is open.      $IMAGE will be replaced
+#. with the image of a hardware menu button.      %1$s will be replaced with
+#. the name of the app (e.g. Firefox for Fire TV)      When this button is
+#. pressed, the overlay will open and the browser will be hidden underneath it
+#, c-format
+msgctxt "hint_press_menu_to_open_overlay"
+msgid "Press $IMAGE for %1$s Home, search, settings, and more"
+msgstr ""
+
+#, c-format
+msgctxt "hint_press_menu_to_open_overlay_a11y"
+msgid "Press the menu button for %1$s Home, search, settings, and more"
```

Comment:
I'm sorry for the back and forth, but we really need to have a comment explaining what the placeholder stands for in each comment.
Our translation tool shows each string individually with each comment. So for example if there are multiple people localizing, not each one will be working on the same strings and seeing their own comments. In the case someone different was working on this string (vs the one before), the person wouldn't know what the placeholder stands for.
thanks!

Reply:
Not a problem!  I assume that means the comment explaining the context (e.g., 'Hint text displayed when the browser is open') should also be duplicated?

Reply:
Yes! thank you 😊